.gitignore

```diff
@@ -19,3 +19,82 @@ springwolf-plugins/springwolf-sqs-plugin/build/
 .idea/
 /gradle.properties
 
+# Eclipse IDE
```

Comment:
I like the eclipse comment and the improvement for Eclipse users.

Are L23-25 could be sufficient? Are the remaining lines needed as well? 

Based on https://labs.consol.de/de/development/git/2017/02/22/gitignore.html I assume that those lines should match recursively.